src/strings/Strings.php

```diff
@@ -0,0 +1,73 @@
+<?php
+
+namespace strings;
+
+class Strings implements StringsInterface
+{
+
+    /**
+     * The $input variable contains text in snake case (i.e. hello_world or this_is_home_task)
+     * Transform it into camel cased string and return (i.e. helloWorld or thisIsHomeTask)
+     * @see http://xahlee.info/comp/camelCase_vs_snake_case.html
+     *
+     * Make sure the next PHPDoc instructions will be added:
+     * @param string $input
+     * @return string
+     */
+    public function snakeCaseToCamelCase(string $input): string
+    {
+        $input = strtolower($input);
```

Comment:
Why are you using `strtolower` here? Snake case cannot contain uppercase letters.